azure-app-service/massdriver.yaml

```diff
@@ -78,16 +78,22 @@ params:
               const: P3v3
         minimum_worker_count:
           title: Minimum instances
-          description: Minimum number of instances for autoscale.
+          description: Minimum number of instances for autoscale. (Enabling zone balancing multiplies this value by 3)
           type: integer
           minimum: 1
           maximum: 30
         maximum_worker_count:
           title: Maximum instances
-          description: Maximum number of instances for autoscale.
+          description: Maximum number of instances for autoscale.(Enabling zone balancing multiplies this value by 3)
           type: integer
           minimum: 1
           maximum: 30
+        zone_balancing:
+          title: Zone balancing
+          description: Enable zone balancing for the App Service Plan. Cannot be changed after deployment. Only supported in East US.
```

Comment:
Only East US supports zone balancing, out of the regions we currently have available for Azure: https://learn.microsoft.com/en-us/azure/availability-zones/migrate-app-service#prerequisites